tests/Fixer/Basic/BracesFixerTest.php

```diff
@@ -4696,21 +4696,21 @@ public function sth(): string
 $foo = new class () extends \Exception {};
 ',
                 null,
-                ['allow_single_line_anonymous_class_with_empty_body' => true],
```

Comment:
Those tests should not be changed: the deprecated option should still work as expected until we actually remove it.

Reply:
ok, I changed it back. But this triggers the `Remaining self deprecation notices` message and fails the tests. I added the `SYMFONY_DEPRECATIONS_HELPER` to `phpunit.xml` help with that. 

Please, let me know if there should be another solution to this.

Reply:
There's a better solution: the test cases using the deprecated option should be moved to a dedicated test method with a `@legacy` annotation and with a `$this->expectDeprecation()` call. You can find examples in other tests in the project.